bunq/sdk/exception_factory.py

```diff
@@ -63,15 +109,21 @@ def _generate_message_error(cls, response_code, messages):
 
         line_response_code = cls._FORMAT_RESPONSE_CODE_LINE \
             .format(response_code)
+        line_response_id = cls._FORMAT_RESPONSE_ID_LINE.format(response_id)
+        line_error_message = cls._FORMAT_ERROR_MESSAGE_LINE.format(
+            cls._GLUE_ERROR_MESSAGE_STRING_EMPTY.join(messages)
```

Comment:
Consider whether `_GLUE_ERROR_MESSAGE` would be sufficient as a name here. Having the name imply it's an empty string is redundant information and not relevant for the consumer IMHO